### PR TITLE
chore: Repo Initial Maintenance

### PR DESCRIPTION
**Pull request by salman2013**
**Description:**

 Initial repo maintenance as per https://docs.openedx.org/en/latest/developers/how-tos/maintain-a-repo.html#how-to-maintain-a-repository

Ticket: https://github.com/orgs/openedx/projects/55/views/1?pane=issue&itemId=41691245

### CODE REVIEW

**Review by feanil — APPROVED** [review]
I think it's fine to leave me in this group but it's fine to remove the rest.

**Comment by salman2013** [comment]
@feanil are you talking about the 'owner' in catalog.yml?

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: "Component"
+metadata:
+  name: 'edx-notes-api'
+  description: "A backend store for edX Student Notes"
+  annotations:
```
Comment (farhan) [inline catalog-info.yaml:9]: We should remove `annotations` key altogether
like
https://github.com/openedx/xblock-lti-consumer/blob/8197943d4e4abc6d058c6038500582df47ed493e/catalog-info.yaml

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+spec:
+
+  # (Required) This can be a group(`group:<group_name>` or a user(`user:<github_username>`)
+  owner: 'group:axim-engineering'
+
+  # (Required) Acceptable Type Values: service, website, library
+  type: 'library'
+
+  # (Required) Acceptable Lifecycle Values: experimental, production, deprecated
+  lifecycle: 'production'
```
Comment (farhan) [inline catalog-info.yaml:24]: Need extra line

**Review by farhan — CHANGES_REQUESTED** [review]
requested changes

**Review by feanil — APPROVED** [review]

**Review by farhan — APPROVED** [review]

**Pull request merged**